### PR TITLE
Remove Ubuntu-ism from text: clarify non-root use

### PR DESCRIPTION
**Pull request by pawsey-kbuckley**
As requested, here is a PR containing some changes that present
a fuller functionality of the `genautocomplete_bash` via its docs.

It's a bit more of a re-write than the changes that the issue I had
raised originally suggested would be needed, but I think I have
"covered all the bases" now.

### CODE REVIEW

**Comment by pawsey-kbuckley** [comment]
Oops,

forgot to explicitly mention the issue that this relates to:  https://github.com/rclone/rclone/issues/7745

File: cmd/genautocomplete/genautocomplete_bash.go [hunk 1]
```diff
@@ -18,20 +18,31 @@ var bashCommandDefinition = &cobra.Command{
...
 
-    sudo rclone genautocomplete bash
+    rclone genautocomplete bash
 
-Logout and login again to use the autocompletion scripts, or source
-them directly
+the generated script will be written to
 
-    . /etc/bash_completion
+/etc/bash_completion.d/rclone
```
Comment (ncw) [inline cmd/genautocomplete/genautocomplete_bash.go:27]: This should probably have 4 spaces at the start so it is rendered as code

File: cmd/genautocomplete/genautocomplete_bash.go [hunk 2]
```diff
@@ -18,20 +18,31 @@ var bashCommandDefinition = &cobra.Command{
...
 
-If you supply a command line argument the script will be written
-there.
+and so will probably need to be run as root, or with sudo.
```
Comment (ncw) [inline cmd/genautocomplete/genautocomplete_bash.go:29]: I think this needs a little more clarification, maybe

    ...and so rclone will probably need to be run as root...

As I wasn't sure whether it was the generated script or rclone that needed running as root reading that.

File: cmd/genautocomplete/genautocomplete_bash.go [hunk 3]
```diff
@@ -18,20 +18,31 @@ var bashCommandDefinition = &cobra.Command{
...
+
+If you supply a path to a file as the command line argument, theh
```
Comment (ncw) [inline cmd/genautocomplete/genautocomplete_bash.go:31]: Typo here `theh`

**Review by ncw — COMMENTED** [review]
This looks great - thank you :-)

I put some minor niggles inline.

Note that if you want this to refer and close the issue then put "Fixes #7745" at the end of the commit message. I can do this when I merge it also if you prefer.

**Comment by pawsey-kbuckley** [comment]
Have fixed up you concerns/notes in my working  copy and pushed,
so my branch should now be ready to merge.

I'll leave the workfllow Fixes addition to you, though.

**Review by ncw — APPROVED** [review]
Thank you :-)

**Pull request merged**